src/test/java/core/basesyntax/ReportServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+    private static Storage storage;
```

Comment:
In your solution Storage is a static map, so you do not need to create a new Storage instance